src/foam/nanos/dig/EmailWebhook.js

```diff
@@ -0,0 +1,26 @@
+/**
+ * @license
+ * Copyright 2022 The FOAM Authors. All Rights Reserved.
+ * http://www.apache.org/licenses/LICENSE-2.0
+ */
+
+foam.INTERFACE({
+  package: 'foam.nanos.dig',
+  name: 'EmailWebhook',
+  documentation: 'Interface for marking an object that contains the externalData map',
+
+  methods: [
```

Comment:
If you add properties it will expand them out to getter and setter methods. If you just want to support getters, then make them methods, like is done here.

Reply:
This is not a service interface, but just something like the various -able interfaces? If so, then yes, make these properties.